pkg/services/ngalert/api/tooling/definitions/time_intervals.go

```diff
@@ -0,0 +1,54 @@
+package definitions
+
+// swagger:route GET /v1/notifications/time-intervals notifications stable RouteNotificationsGetTimeIntervals
+//
+// Get all the time intervals
+//
+//     Responses:
+//       200: NamedTimeIntervalsListWithProvenance
+//       403: ForbiddenError
+
+// swagger:route GET /v1/notifications/time-intervals/{name} notifications stable RouteNotificationsGetTimeInterval
+//
+// Get a time interval by name.
+//
+//     Responses:
+//       200: NamedTimeIntervalsWithProvenance
+//       404: NotFound
+//       403: ForbiddenError
+
+// swagger:parameters stable RouteNotificationsGetTimeInterval
+type RouteTimeIntervalNameParam struct {
+	// Time interval name
+	// in:path
+	Name string `json:"name"`
+}
+
+// swagger:model
+type NamedTimeIntervals struct {
```

Comment:
WDYT about continuing to use gettable/postable prefixes to differentiate between POST and GET schemas? It leaves us with more leeway to alter the type without the name drifting from what's represented.

Reply:
I was on the fence about that as Postalbe\Gettable from specs perspective look weird. However, after you asked I tried it and I kinda ok with it.